src/App/utils/resourcesByType/index.js

```diff
@@ -610,7 +610,8 @@ export const resourcesByType = {
           'onError': 'func',
           'method': requestMethods,
           'subscribe': 'bool',
-          'subscribeInterval': 'number'
+          'subscribeInterval': 'number',
+          'showLoadingBetweenRequests': 'bool',
```

Comment:
New option on `Request` if you want the placeholder/loading indicator to show between requests after the first; by default, the placeholder/loading will show only on the first request (so that you can poll or still work without the user knowing there is more data loading, but if data is required this option forces the UI to show a placeholder/loading state until the new data is ready)